src/main/java/org/example/dao/TicketDaoImpl.java

```diff
@@ -12,12 +13,19 @@
 import java.util.ArrayList;
 import java.util.UUID;
 
-public class TicketDaoImpl implements TicketDao {
+@Component
```

Comment:
for DAO level it is better to use specific annotation: `@Repository`

Reply:
Changed annotation from @Component to @Repository